### PR TITLE
update expected output for qwen2

### PR DESCRIPTION
**Pull request by nprotasov**
Update expected output for qwen2 model: current output looks wrong and was generated from main branch, there was error in computing logits and loss.
New output looks more correct

### CODE REVIEW

**Comment by regisss** [comment]
@nprotasov Why targeting the branch `integ_future_01312025` and not `transformers_future`?
